src/main/resources/application.yml

```diff
@@ -8,7 +8,7 @@ server.port: 8080
 spring:
   ### Database ###
   datasource:
-    url: "jdbc:h2:file:./h2/suricate;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE"
+    url: "jdbc:h2:file:./h2/suricate;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE;MODE=PostgreSQL"
```

Comment:
Will it work with MySQL ??

Reply:
Because i have set Postgre as default migration folder below.

Reply:
Can we use spring profiles to have both available ?